keras/backend/tensorflow_backend.py

```diff
@@ -402,7 +402,7 @@ def __call__(self, inputs):
         return updated[:len(self.outputs)]
 
 
-def function(inputs, outputs, updates=[]):
+def function(inputs, outputs, updates=[], **kwargs):
```

Comment:
Since no arguments are expected, please add a check that none are passed (with helpful error message otherwise).